openreview/journal/invitation.py

```diff
@@ -6433,6 +6434,69 @@ def set_note_reviewer_message_invitation(self, note):
             signatures=[self.journal.venue_id]
         )
 
+    def set_preferred_emails_invitation(self):
+
+        venue_id = self.journal.venue_id
+
+        if openreview.tools.get_invitation(self.client, self.journal.get_preferred_emails_invitation_id()):
+            return
+
+        invitation = Invitation(
+            id=self.journal.get_preferred_emails_invitation_id(),
+            invitees=[venue_id],
+            readers=[venue_id],
+            writers=[venue_id],
+            signatures=['~Super_User1'], ## it should be the super user to get full email addresses
+            minReplies=1,
+            maxReplies=1,
+            type='Edge',
+            edit={
```

Comment:
Is this how it works in openreview-py?

Reply:
yeah, should I do it in a different way?

Reply:
Well, not in this PR, but I thought that it would be like in the API where we pass `edge` instead of `edit` when it's an edge.

Reply:
we do that when serialize the invitation: https://github.com/openreview/openreview-py/blob/master/openreview/api/client.py#L2725